### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by txtsd**
Here are some low-hanging fruits I picked.

The commit messages are pretty descriptive.

The typo changes have been submitted upstream too:
[grumdrig/pq](https://bitbucket.org/grumdrig/pq/pull-requests/1) (Merged)
[grumdrig/pq-web](https://bitbucket.org/grumdrig/pq-web/pull-requests/1/lots-of-fixes-for-the-codebase) ~~(Awaiting review)~~ (Merged)

### CODE REVIEW

File: pqcli/ui/curses/views/game_view/quest_book_window.py [hunk 1]
```diff
@@ -46,7 +46,7 @@ def sync(self) -> None:
 
     def _sync_quests(self) -> None:
         self._list_box.clear()
-        for quest_name in self._player.quest_book.quests[-self.cutoff :]:
+        for quest_name in self._player.quest_book.quests[-self.cutoff:]:
```
Comment (rr-) [inline pqcli/ui/curses/views/game_view/quest_book_window.py:49]: Please revert.

File: pqcli/ui/curses/widgets/list_box.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def get(self, idx: int) -> T.Optional[str]:
             return None
 
     def delete(self, idx: int, count: int = 1) -> None:
-        del self._items[idx : idx + count]
+        del self._items[idx: idx + count]
```
Comment (rr-) [inline pqcli/ui/curses/widgets/list_box.py:36]: Please revert.

**Review by rr- — COMMENTED** [review]
Thanks for the pull request and meticulous typo fixes.
A couple of notes:
- commit names do not adhere to the repo's convention
- some of the code changes violate our style guides (pre-commit, specifically Black)

File: pqcli/ui/curses/views/game_view/quest_book_window.py (hunk 1 above)
Comment (txtsd) [reply pqcli/ui/curses/views/game_view/quest_book_window.py]: Done

File: pqcli/ui/curses/widgets/list_box.py (hunk 2 above)
Comment (txtsd) [reply pqcli/ui/curses/widgets/list_box.py]: Done

**Comment by txtsd** [comment]
Oops. I had forgotten to `poetry run install pre-commit`.

What is the name of the commit message convention? I use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

**Review by rr- — APPROVED** [review]

**Comment by txtsd** [comment]
Thanks!

**Pull request merged**